gradle.properties

```diff
@@ -1,7 +1,7 @@
 org.gradle.caching=true
 group=io.ballerina.lib
-version=1.0.1-SNAPSHOT
-ballerinaLangVersion=2201.9.0
+version=1.1.1-SNAPSHOT
+ballerinaLangVersion=2201.10.0
```

Comment:
do we need to bump the lang version?

Reply:
Since this is required for lang 9.4 release, I will change the version back to 2201.9.0